src/System.Runtime.Extensions/src/System/IO/Path.Unix.cs

```diff
@@ -205,6 +206,31 @@ public static string GetPathRoot(string path)
 
         private static unsafe void GetCryptoRandomBytes(byte* bytes, int byteCount)
         {
+            if (Environment.IsMac)
+            {
+                GetCryptoRandomBytesApple(bytes, byteCount);
+            }
+            else
+            {
+                GetCryptoRandomBytesOpenSsl(bytes, byteCount);
+            }
+        }
+
+        private static unsafe void GetCryptoRandomBytesApple(byte* bytes, int byteCount)
+        {
+            Debug.Assert(bytes != null);
+            Debug.Assert(byteCount >= 0);
+
+            int result = CCRandomGenerateBytes(bytes, byteCount);
```

Comment:
I could see that being a problem; and the solution would seem to be splitting the compile.  I had done that originally, but ran into some complications with keeping a "unix" and "osx" build (so that FreeBSD/etc still worked) because this comes down as a thin package.  Eric told me some ways to work around it, but this felt cleaner.

So, to keep packaging easy, I'd like to hold off the split build until the need is proven.